src/components/project-tabs/QRTab.jsx

```diff
@@ -1,26 +1,90 @@
-import React, { Component } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import QRCode from "qrcode.react";
+import { FeedbackContext } from "../../contexts/feedback-context";
 import "../../css/project-tabs/QRtab.css";
+import Loader from "react-loader-spinner";
 import "react-bootstrap";
+import moment from 'moment';
+import api from "../../api.js";
+import * as Icon from 'react-bootstrap-icons';
 
-class QRTab extends Component {
-  // eslint-disable-next-line no-useless-constructor
-  constructor(props) {
-    super(props);
+const QRTab = (props) => {
+  // useContext can be used to access global context and dispatch changes
+  const [state, dispatch] = useContext(FeedbackContext);
+
+  useEffect(() => {
+    const param = props.projectID;
+    api
+      .get(`/projects/${param}/conversions`)
+      .then((res) => {
+        if (res.status == 200 && res.data.length > 0) {
+          dispatch({
+            type: "SET_CONVERSIONS",
+            payload: res.data
+          });
+        }
+      })
+      .catch((err) => { console.log(err) })
+  }, [props.projectID]);
+
+  // Sort conversion from latest to oldest (descending order)
+  const sortByTimestamp = function (a, b) {
+    return b.timestamp - a.timestamp;
+  };
+  var conversions = [];
+  var lastConversion = {};
+  if (!!state.conversions) {
+    conversions = state.conversions.sort(sortByTimestamp);
+    lastConversion = conversions[0];
   }
 
-  render() {
+  // Create QR code link
+  const qrCodeLink = `exp://exp.host/@imaginethis/${state.projectID}`;
+
+  // Depending on the status of last conversion show different contents
+  if (!lastConversion) {
+    return (
+      <div style={{display: "flex", justifyContent: "center", alignItems: "center", height: 200}}>
+        <Icon.Box color="#005EB8" size={70} />
```

Comment:
I think we should try to avoid using inline css, normally we try to separate this stuff to its own css file.

Reply:
done